File: cpp/1980.cpp
class Solution {
public:
    string findDifferentBinaryString(vector<string>& nums) {
        int n = nums.size();
        string res = "";
        for (int i = 0; i < n; i++)
            res += (nums[i][i] == '1') ? '0' : '1';
        return res;
    }
};

File: java/1980.java
class Solution {
    public String findDifferentBinaryString(String[] nums) {
        int n = nums.length;
        char res[] = new char[n];
        for (int i = 0; i < n; i++)
            res[i] = (nums[i].charAt(i) == '1') ? '0' : '1';
        return new String(res);
    }
}
